docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "FreeFunctionAliasing",
```

Comment:
Maybe not focus too much on aliasing?

Reply:
Revert to `FreeFunctionRedfinition` for the bug list? Other ideas `FreeFunction`, `FreeFunctionNoError`, `FreeFunctionIdentical`. Wdyt?

Reply:
name clash? redefinition?